src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -96,6 +99,42 @@ More information about these app properties, the types they
 depend on, and how internationalized Flutter apps are typically
 structured, can be found below.
 
+<a name="advanced-locale"></a>
+## Advanced locale definition
+
+Some languages with multiple variants require more than just a language code to
+properly differentiate.
+
+For example, fully differentiating all variants of Chinese requires specifying
+the language code, script code, and country code. This is due to the existence
+of simplified and traditional script, as well as regional differences in the way
+characters are written within the same script type.
+
+In order to fully express every variant of Chinese for the country codes `CN`,
+`TW`, and `HK`, the list of supported locales should include:
+
+{% prettify dart %}
+// Full Chinese support for CN, TW, and HK
+supportedLocales: [
+  const Locale.fromSubtags(languageCode: 'zh'), // generic Chinese 'zh'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hans'), // generic simplified Chinese 'zh_Hans'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hant'), // generic traditional Chinese 'zh_Hant'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hans', countryCode: ), // 'zh_Hans_CN'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hant', countryCode: ), // 'zh_Hant_TW'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hant', countryCode: ), // 'zh_Hant_HK'
+],
+{% endprettify %}
+
+This explicit full definition will ensure your app can distinguish between and provide
+the fully nuanced localized content to all combinations of these country codes. If a
+user's preferred locale is not specified, then the closest match will be used instead,
+which will likely contain differences to what the user expects. See
+[`Localizations`]({{site.api}}/flutter/widgets/WidgetsApp/supportedLocales.html) for
+how the supported locales and the preferred locales are resolved.
+
+Although Chinese is a primary example, other languages like French (FR_fr, FR_ca, etc)
```

Comment:
Do we have a comprehensive list of supported locales that would benefit from being "fully differentiated"?

Reply:
We may be able to throw one together, but the total combinations of all possible countries/tags/etc may be extremely large and not particularly useful. This basically applies to anything with more than just a language code, but only becomes relevant when the variations are significant.